src/path.cpp

```diff
@@ -165,6 +165,42 @@ namespace {
 		// convert to seconds
 		return time_t(ft / 10000000 - posix_time_offset);
 	}
+
+	FILETIME large_integer_to_file_time(LARGE_INTEGER i)
```

Comment:
since `LARGE_INTEGER` is already part of the signature (you can't pass anything else to it), I think it would be fine for this function to just be called `to_file_time()`.